src/commands/stac-validate/stac.validate.ts

```diff
@@ -268,7 +268,19 @@ export function getStacChildren(stacJson: st.StacItem | st.StacCollection | st.S
 }
 
 export function normaliseHref(href: string, path: string): string {
-  return new URL(href, path).href;
+  if (isURL(path)) {
+    return new URL(href, path).href;
+  }
+  return join(path.substring(0, path.lastIndexOf('/')), href);
```

Comment:
`dirname` is  safer than sub stringing and looking for `/` (especially for windows users)